update hardcoded repos and README (#55)
Imageomics/repo-exporter: README.md

 8. Go to https://console.cloud.google.com/apis/library/sheets.googleapis.com and enable the **Google Sheets API** for the project you made
 9. Go to your chosen Google Sheet and go to **Share** settings and add the new Service Account email you made and set it as an **Editor**

After forking the repository, configure the environment variables required for the exporter(s) you plan to run.

### GitHub exporter

- `GH_ORG_NAME` — GitHub organization name
- `GH_SPREADSHEET_ID` — Google Sheet ID for GitHub export data
- `GH_SHEET_NAME` — worksheet tab used by the GitHub exporter
- `GH_TOKEN` — GitHub access token

### Hugging Face exporter

- `HF_ORG_NAME` — Hugging Face organization name
- `HF_SPREADSHEET_ID` — Google Sheet ID for Hugging Face export data
- `HF_SHEET_NAME` — worksheet tab used by the Hugging Face exporter
- `HF_TOKEN` — Hugging Face access token

### Shared

- `GOOGLE_CREDENTIALS_PATH` — path to the Google service account JSON file when running locally

> - `GOOGLE_SERVICE_ACCOUNT_JSON` — GitHub Actions secret containing the service account JSON

Reviewer: ```suggestion
After cloning the repository, configure the [environment variables](#environment-variables) required for the exporter(s) you plan to run.
```
This is repetitive, so linking to a new section below.

    cd repo-exporter
    ```

2. Create and activate the Conda environment:
   ```
   conda create -n repo-exporter python -y
   conda activate repo-exporter
   ```
    
3. Configure only the variables required for the exporter(s) you plan to run.

```bash
# GitHub exporter variables
conda env config vars set GH_TOKEN="<your-token-here>"
conda env config vars set GH_ORG_NAME="<your-github-org>"
conda env config vars set GH_SPREADSHEET_ID="<your-google-sheet-id>"
conda env config vars set GH_SHEET_NAME="<your-sheet-name>"

# Hugging Face exporter variables
conda env config vars set HF_TOKEN="<your-huggingface-token-here>"
conda env config vars set HF_ORG_NAME="<your-huggingface-org>"
conda env config vars set HF_SPREADSHEET_ID="<your-google-sheet-id>"
conda env config vars set HF_SHEET_NAME="<your-sheet-name>"

# Shared variables

> conda env config vars set GOOGLE_CREDENTIALS_PATH="/path/to/service_account.json"

Reviewer: We should really create a `.env` for all of these values (especially the tokens, which must be secrets). That should then be added to the `.gitignore` if it's not already. We _can_ include something like `.env-example` with all the values listed like this. Then we note that for GitHub use, the tokens should be saved under secrets and variables under repo variables.

Author: Sorry, just saw this! Will work on it.

    cd repo-exporter
    ```

2. Create and activate the Conda environment:
   ```
   conda create -n repo-exporter python -y
   conda activate repo-exporter
   ```
    
3. Configure environment variables using a `.env` file

Create a `.env` file in the root of the project:

```bash
GH_TOKEN=your-token-here
GH_ORG_NAME=your-github-org
SPREADSHEET_ID=your-google-sheet-id
GH_SHEET_NAME=GH-Repos

HF_TOKEN=your-huggingface-token
HF_ORG_NAME=your-huggingface-org
HF_SHEET_NAME=HF-Repos

GOOGLE_CREDENTIALS_PATH=/path/to/service_account.json

> ```

Reviewer: ```suggestion
3. Create a `.env` file in the root of the project to configure required [environment variables](#environment-variables). See [`.env.example`](.env.example) for an example; the default sheet names are included and can be removed.
```

> [!NOTE]
> `SPREADSHEET_ID` and Google service account credentials are required for both GitHub and Hugging Face exports, as well as the platform-prefaced variables (e.g., `GH_ORG_NAME` and `HF_ORG_NAME`)
> * Both exporters support optional custom sheet name variables (`GH_SHEET_NAME` and `HF_SHEET_NAME`; these must be *distinct*).

> [!WARNING]  
> Tokens and Google service account credentials must be saved as repository *secrets*. All other values may be saved as environment variables assuming your spreadsheet is appropriately protected.

### GitHub exporter

* Set `GH_ORG_NAME` to your GitHub organization name (for API calls).
* Set `SPREADSHEET_ID` to the Google Sheet ID used by the exporter.
* `GH_SHEET_NAME` is optional. If not provided, the exporter uses "GH-Repos".
* `GH_TOKEN` is required to access GitHub repositories.

### Hugging Face exporter

* Set `HF_ORG_NAME` to your Hugging Face organization name (**case-sensitive**, for API calls).
* Set `SPREADSHEET_ID` to the Google Sheet ID used by the exporter.
* `HF_SHEET_NAME` is optional. If not provided, the exporter uses "HF-Repos".
* `HF_TOKEN` is required to access Hugging Face repositories.

### Shared configuration

> * Both exporters require `GOOGLE_CREDENTIALS_PATH`.

Author: This makes sense, but I'm not sure if you prefer this way of wording. What are your thoughts on it?

> * Both exporters require `GOOGLE_CREDENTIALS_PATH`.

Reviewer: A slight change to the suggestion to provide a reference:
```
Both exporters require Google service account credentials created in the [service account setup](#set-up-google-cloud-service-account-access). Set `GOOGLE_CREDENTIALS_PATH` to the JSON key file path (defaults to `service_account.json` if omitted).
```

      python gh_repo_exporter.py
      ```

## Environment Variables

> [!NOTE]
> * `gh_repo_exporter.py` requires the `GH_*` and `SPREADSHEET_ID`. 
> * `hf_repo_exporter.py` requires the `HF_*` and `SPREADSHEET_ID`. 

> > * Both exporters support optional sheet name variables (`GH_SHEET_NAME` and `HF_SHEET_NAME`) and require Google service account credentials.

Reviewer: ```suggestion
> `SPREADSHEET_ID` and Google service account credentials are required for both GitHub and Hugging Face exports, as well as the platform-prefaced variables (e.g., GH_ORG_NAME` and `HF_ORG_NAME`)
> * Both exporters support optional custom sheet name variables (`GH_SHEET_NAME` and `HF_SHEET_NAME`; these must be *distinct*).

> [!WARNING]  
> Tokens and Google service account credentials must be saved as repository *secrets*. All other values may be saved as environment variables assuming your spreadsheet is appropriately protected.
```
Suggestion to try to make it a bit more consolidated/less repetitive, but let me know if it doesn't make sense.

Also, clarifying secret vs variables.

Author: Thank you! Could you touch more on how to make READMEs more consolidated? This is my first time working on a README.md so I don't know much on its standard practices and stuff.

Reviewer: We point to a [couple examples](https://imageomics.github.io/Imageomics-guide/wiki-guide/GitHub-Repo-Guide/#examples) in the Guide (under the [general README guidance](https://imageomics.github.io/Imageomics-guide/wiki-guide/GitHub-Repo-Guide/#readme)). It's usually good to have a summary at the top and get more detailed further down, especially since those sections can be referenced. Every repo is different, but it's nice when it flows with how a user would interact with the repo, ex:
1. What is this repo?
2. How do I install the software?
3. What are some example uses?
4. Testing/development guidance
5. References/acknowledgments

Author: Thank you! I forgot that theres a README section in the repo guide. I will take a look into that.

### GitHub exporter

* Set `GH_ORG_NAME` to your GitHub organization name (for API calls).
* Set `SPREADSHEET_ID` to the Google Sheet ID used by the exporter.
* `GH_SHEET_NAME` is optional. If not provided, the exporter uses "GH-Repos".
* `GH_TOKEN` is required to access GitHub repositories.

### Hugging Face exporter

* Set `HF_ORG_NAME` to your Hugging Face organization name (**case-sensitive**, for API calls).
* Set `SPREADSHEET_ID` to the Google Sheet ID used by the exporter.
* `HF_SHEET_NAME` is optional. If not provided, the exporter uses "HF-Repos".
* `HF_TOKEN` is required to access Hugging Face repositories.

### Shared configuration

* Both exporters require `GOOGLE_CREDENTIALS_PATH`.
* The Google service account must have Editor access to the target spreadsheet.

> * If both exporters write to the same spreadsheet or worksheet, you may reuse the same spreadsheet IDs and sheet names.

Reviewer: ```suggestion
```
They will write to the same spreadsheet, but must have different sheets in it.

### GitHub exporter

* Set `GH_ORG_NAME` to your GitHub organization name (for API calls).
* Set `GH_SPREADSHEET_ID` to the Google Sheet ID used by the GitHub exporter.
* Set `GH_SHEET_NAME` to the worksheet tab used by the GitHub exporter.
* `GH_TOKEN` is required to access GitHub repositories.

### Hugging Face exporter

* Set `HF_ORG_NAME` to your Hugging Face organization name (**case-sensitive**, for API calls).
* Set `HF_SPREADSHEET_ID` to the Google Sheet ID used by the Hugging Face exporter.
* Set `HF_SHEET_NAME` to the worksheet tab used by the Hugging Face exporter.
* `HF_TOKEN` is required to access Hugging Face repositories.

### Shared configuration

* Both exporters require `GOOGLE_CREDENTIALS_PATH`.
* The Google service account must have Editor access to the target spreadsheet.
* If both exporters write to the same spreadsheet or worksheet, you may reuse the same spreadsheet IDs and sheet names.
* Ensure all required values are available as environment variables locally or as GitHub Actions secrets.

For example, if the spreadsheet URL is:

```text id="1i7ltb"
https://docs.google.com/spreadsheets/d/fake-long-alpha-numeric-id-1a2b3c4d/edit

> ```

Reviewer: ````suggestion
```

then the spreadsheet ID is:

```text id="04f0vl"
fake-long-alpha-numeric-id-1a2b3c4d
```
````
something went funny applying this change

> ## Important Notes

Reviewer: ```suggestion
## Environment Variables
```

* `gh_repo_exporter.py` only requires the `GH_*` environment variables.
* `hf_repo_exporter.py` only requires the `HF_*` environment variables.

> * Both exporters require Google service account credentials.

Reviewer: ```suggestion
> [!NOTE]
> * `gh_repo_exporter.py` only requires the `GH_*` environment variables.
> * `hf_repo_exporter.py` only requires the `HF_*` environment variables.
> * Both exporters require Google service account credentials.
```

### GitHub exporter

> * Set `GH_ORG_NAME` to your GitHub organization name.

Reviewer: ```suggestion
* Set `GH_ORG_NAME` to your GitHub organization name (for API calls).
```

* Set `GH_SPREADSHEET_ID` to the Google Sheet ID used by the GitHub exporter.
* Set `GH_SHEET_NAME` to the worksheet tab used by the GitHub exporter.
* `GH_TOKEN` is required to access GitHub repositories.

### Hugging Face exporter

> * Set `HF_ORG_NAME` to your Hugging Face organization name.

Reviewer: ```suggestion
* Set `HF_ORG_NAME` to your Hugging Face organization name (**case-sensitive**, for API calls).
```

* Set `HF_SPREADSHEET_ID` to the Google Sheet ID used by the Hugging Face exporter.
* Set `HF_SHEET_NAME` to the worksheet tab used by the Hugging Face exporter.
* `HF_TOKEN` is required to access Hugging Face repositories.

### Shared configuration

* Both exporters require `GOOGLE_CREDENTIALS_PATH`.
* The Google service account must have Editor access to the target spreadsheet.
* If both exporters write to the same spreadsheet or worksheet, you may reuse the same spreadsheet IDs and sheet names.
* Ensure all required values are available as environment variables locally or as GitHub Actions secrets.

For example, if the spreadsheet URL is:

```text id="1i7ltb"
https://docs.google.com/spreadsheets/d/15BQimTjaOyo-jeaJRcg1Hia-9ORcilj3Jx-ks-uGyoc/edit
```

then the spreadsheet ID is:

```text id="04f0vl"

> 15BQimTjaOyo-jeaJRcg1Hia-9ORcilj3Jx-ks-uGyoc

Reviewer: ````suggestion
https://docs.google.com/spreadsheets/d/fake-long-alpha-numeric-id-1a2b3c4d/edit
```

then the spreadsheet ID is:

```text id="04f0vl"
fake-long-alpha-numeric-id-1a2b3c4d
````
